app/models/DefaultPromos.scala

```diff
@@ -0,0 +1,14 @@
+package models
+
+import io.circe.generic.auto._
+import io.circe.{Decoder, Encoder}
+
+case class DefaultPromos(
+                          guardianWeekly: Seq[String],
+                          paper: Seq[String]
+                               )
```

Comment:
The formatting looks slightly out here. It doesn't look like scalafmt is configured for this project but perhaps we could fix manually?

Reply:
I've fixed this manually, should be good now.